docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -223,9 +223,6 @@ Preparing For a Release
 
    Wait for an accepted code review before continuing.
 
-   .. warning:: Add a note to the pull request description explaining that the branch should not be merged until the release process is complete.
-
-
```

Comment:
They are in that manual testing of this part of the process was the last remaining work on the post-release review, so I removed it and moved the merge earlier. This is explained at https://github.com/ClusterHQ/flocker/pull/1542#discussion_r31993933.